eq-author/src/App/QuestionnairesPage/QuestionnairesTable/index.test.js

```diff
@@ -157,6 +158,49 @@ describe("QuestionnairesTable", () => {
     ).toMatchObject({ autoFocus: true });
   });
 
+  it("should show delete question confirmation when modal confirm clicked", () => {
```

Comment:
This test is fully covered in the test below. Maybe we could remove this one?

Reply:
Agreed. I thought the same but I just transposed existing tests from Row to QuestionnaireTable so wasn't sure if there was a reason for it.